templates/docs/content-guidelines.md

```diff
@@ -0,0 +1,190 @@
+- **Actions:** Often verbs, things users can do with objects.
+
+<div class="u-fixed-width"><img src="https://assets.ubuntu.com/v1/cde94563-lxd-context.png" alt="A capture of a form of the LXD UI, including the app side navigation. The context relevant for an error message is outlined on the screenshot."></div>
+
+We should be specific about objects and actions and not have duplicate terms. Our messages typically include three attributes:
+
+- **Title**
+- **Description**
+- **CTA**
+
+**Example from Anbox:**
+
+<div class="u-fixed-width"><img src="https://assets.ubuntu.com/v1/e53e5133-reset-dialog.png" alt="Example of a reset dialog in Anbox with contextualized title, description and CTA."></div>
+
+You may need to consult with your product manager or engineers to clarify the objects and actions in different circumstances. Follow the guidance below to write messages that support users.
+
+### Title
+
+- Use a verb phrase to describe the action that will affect a certain object. In the example above, the action is to ‘reset’ and the object is ‘default values’.
+- Do not use punctuation in the title.
+
+### Description
+
+- Add details about the action, try not to add more objects.
+- Use punctuation in the description.
+
+**Research says to:**
+
+- Aim for 3-6 words per line, no more than 3 lines per message.
+- Avoid asterisks (users don’t really trust them).
+
+### CTAs
+
+- Match the text of the primary CTA button to the action in the title.
+- Note: our current convention is to make ‘Cancel’ the secondary CTA button. It performs the same action as an ‘X’ icon in the top right corner.
+
+## Notification library
+
+The following is a library of messages you can use or edit to your specific needs.
+
+**Error: Unspecified systems error**  
+Use this kind of message when we don’t know what caused the error. Adjust the message to match the action the system can’t complete and the options the user has to fix the issue.
+
+**Installation failed**  
+We're sorry, but we're not sure what went wrong. You can try restarting your computer and start the installation process again. You can also report the issue.
+
+**Error: No connection**  
+Connect to the internet  
+We can't load [insert specifics of what we cannot load] without an internet connection. Check your connection and reload.
+
+**Example from App Store:**  
+We can't load content in the App Center without an internet connection. Check your connection and reload.
+
+**Error: 404 for Vanilla**  
+404 Page not found  
+We can’t find the page you’re looking for. File a bug if you think this might be an error.
+
+<div class="u-fixed-width"><img src="https://assets.ubuntu.com/v1/98071afb-page-not-found.png" alt="Example of a 404 page from Vanilla"></div>
+
+**Confirmation: Contact form submission**  
+‘Thank you for contacting us. A member of our team will be in touch shortly.’
+
+> Note: Follow the guidance as described for consistency and accessibility across all content.
```

Comment:
I think this (and possibly the "notification" examples above, could work best as notification component).

Somewhat like we do in other places in the docs:
https://github.com/canonical/vanilla-framework/blob/main/templates/docs/base/forms.md?plain=1#L161-L166

On site: https://vanillaframework.io/docs/base/forms#disabled